facefusion/face_helper.py

```diff
@@ -53,6 +53,30 @@ def paste_back(temp_frame : Frame, crop_frame : Frame, affine_matrix : Matrix) -
 	return temp_frame
 
 
+def paste_back(background: numpy.ndarray, foreground: numpy.ndarray, affine_matrix: numpy.ndarray, fade_amount: float = 0.25, offset_factor: Tuple[float, float, float, float] = ( 0.0, 0.0, 0.0, 0.0 )) -> Frame:
```

Comment:
Please keep the naming and typing of the project

```
paste_back_old(temp_frame : Frame, crop_frame : Frame, affine_matrix : Matrix, blur_size : float, padding : Padding) -> Frame:
```

Add new Padding to typing.py:

```
Padding : Tuple[float, float, float, float]
```

Do not use defaults in for method parameter, this is considered a bad practice as it will add magic behaviour all over the place - it's better to enforce the person using a method, to specify the defaults in the call